robotpy_build/platforms.py

```diff
@@ -62,7 +62,11 @@ def get_platform(name: typing.Optional[str] = None) -> WPILibMavenPlatform:
         pyplatform = _get_platform()
 
         # Check for 64 bit x86 macOS (version agnostic)
-        if re.fullmatch(r"macosx-.*-x86_64", pyplatform):
+        # - See https://github.com/pypa/setuptools/issues/2520 for universal2
```

Comment:
This is almost certainly going to lead to problems, but not sure how to deal with it for now.

Reply:
We should probably look into supporting native ARM at some point, iirc WPILib is building for ARM for next season